lib/internal/process/next_tick.js

```diff
@@ -315,13 +315,13 @@ function setupNextTick() {
     }
 
     const asyncId = ++async_uid_fields[kAsyncUidCntr];
+    if (triggerAsyncId === null) {
```

Comment:
Nit: Since it's argument overload handling, could you move it up, probably just after the `if (process._exiting) return`